### PR TITLE
Feature: social media login

### PR DESCRIPTION
**Pull request by Abdi-Tiruneh**
**The following feature is added**

- facebook login

1. google login

### CODE REVIEW

File: file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello
```
Comment (Abdi-Tiruneh2) [inline file.txt:1]: this line is good. but ist better if you make h capital

File: file1.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+hello
```
Comment (Abdi-Tiruneh2) [inline file1.txt:1]: make it capita

**Review by Abdi-Tiruneh2 — CHANGES_REQUESTED** [review]
change what said and requist agai

File: file.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Hello
```
Comment (Abdi-Tiruneh2) [inline file.txt:1]: good

File: file.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+my name is
```
Comment (Abdi-Tiruneh2) [inline file.txt:2]: perfect

**Review by Abdi-Tiruneh2 — APPROVED** [review]
perfect

**Comment by Abdi-Tiruneh** [comment]
**Good boy**

**Pull request merged**